pkg/oci/build.go

```diff
@@ -204,13 +206,19 @@ func createManifestForTarget(ctx context.Context, target oras.Target, metadataFi
 		if err != nil {
 			return ocispec.Descriptor{}, fmt.Errorf("creating metadata descriptor: %w", err)
 		}
+		defDesc.Annotations[ocispec.AnnotationCreated] = createdDate
```

Comment:
Now that we are setting created annotation using `createdDate` do we still need https://github.com/inspektor-gadget/inspektor-gadget/blob/main/pkg/oci/build.go#L150 ?